openslides_backend/action/motion/reset_state.py

```diff
@@ -0,0 +1,51 @@
+from typing import Any, Dict
+
+from ...models.models import Motion
+from ...shared.exceptions import ActionException
+from ...shared.patterns import Collection, FullQualifiedId
+from ...shared.schema import schema_version
+from ..generics import UpdateAction
+from ..register import register_action
+
+reset_schema = {
```

Comment:
Please just use the default update schema without any properties.

Reply:
Done. Use default update schema.